platform/kube/client.go

```diff
@@ -232,7 +232,8 @@ func (cl *Client) Get(key model.Key) (proto.Message, bool) {
 		return nil, false
 	}
 
-	out, err := mapToProto(cl.mapping[key.Kind].MessageName, config.Spec)
+	kind := cl.mapping[key.Kind]
+	out, err := kind.FromJSONMap(config.Spec)
 	if err != nil {
```

Comment:
We need protobuf specific conversion stuff in `model/conversion.go`. `kube/conversion.go` is for k8s